terraform/gcp/featureform/featureform.tf

```diff
@@ -35,6 +35,16 @@ resource "helm_release" "featureform" {
     value = var.featureform_public_cert
   }
 
+  set {
+    name = "global.publicCert"
+    value = "true"
```

Comment:
would make localCert the default one so it matches with the rest of the quickstarts